src/databricks/labs/ucx/source_code/notebooks/commands.py

```diff
@@ -0,0 +1,48 @@
+import re
+import shlex
+from typing import TypeVar, cast
+
+from databricks.labs.ucx.source_code.graph import DependencyGraph, DependencyProblem
+
+T = TypeVar("T")
+
+
+class PipCommand:
```

Comment:
```suggestion
class PipMagic:
```
command is something different in the Jupyter protocol.

Reply:
done